www/kadv05/sp2/pokemon-idle/src/components/OpponentSection.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+const OpponentSection = ({ currentOpponent, onOpponentClick, formatNumber, capitalizeFirstLetter }) => {
+  const calculateHealthPercentage = () => {
+    if (currentOpponent.maxHP === 0) {
+      return 0;
+    }
+    return (currentOpponent.hp / currentOpponent.maxHP) * 100;

```

Comment:
![image](https://github.com/nvbach91/4IZ268-2023-2024-ZS/assets/20724910/8ebb532b-7739-4001-9c60-024bc6d35f4c)
co to jsou za cisla?

Reply:
hp a maxHP (jsou vynásobeny multiplikátorem pro škálování obtížnosti)